Gruntfile.js

```diff
@@ -42,7 +43,7 @@ module.exports = function (grunt) {
     }
 
     // Default task
-    grunt.registerTask('default', ['clean', 'validate', 'prepare', 'compile', 'test', 'analyse']);
+    grunt.registerTask('default', ['install', 'clean', 'validate', 'compile', 'test', 'analyse']);
```

Comment:
ha! just removes any packages still lying around that aren't required anymore. like `npm prune`